### DISCUSSION TITLE
API endpoint extension

### DISCUSSION THREAD

**Discussion by worldofjimmy** [AscensionGameDev/Intersect-Engine#1508] [Ideas]
**Is your feature request related to a problem? Please describe.**:
Sort Users by Player/Global Variable, would open up possibilities to create rankings in custom created skills stored in Player/Global Variables.

**Describe the solution you'd like**:
An API endpoint to retrieve data of Users with their Player/Global variable value and sort this in order to create custom rankings in example, PVP, Cooking, Mining, quest points etc.

**Additional context**:
Endless possibilities with this API endpoint for website builders.

**Comment by pandinocoder** [comment]
Normally I would have left this as an issue but this is going to be much more complicated to accomplish under the current architecture than is reasonable to leave as slated for a release. If this gets sufficient upvotes this can be revisited.
